examples/rl1/subset.py

```diff
@@ -33,7 +33,7 @@ def go(inpfn, firstpage, lastpage):
     canvas = Canvas(outfn)
 
     for page in pages:
-        canvas.setPageSize(tuple(page.BBox[2:]))
+        canvas.setPageSize((page.BBox[2], page.BBox[3]))
```

Comment:
This is an interesting change -- was it perhaps in response to a bad PDF with a bbox with more than 4 elements?